devicon.json

```
@@ -2140,6 +2140,30 @@
         "color": "#5E5185",
         "aliases": []
     },
+    
+    {
+       "name": "hadoop",
+       "altnames": [],
+       "tags": [
+	  "framework",
+	  "big data"
+  	],
+  	"versions": {
+	    "svg": [ 
+	      "original",
+	      "original-wordmark",
+	      "line"
+	    ],
+	    "font": [ 
+	      "original",
+	      "original-wordmark"
+	    ]
+	  },
+  	"color": "#fdeb4f", 
+  	"aliases": [] 
```

Hmm, do you think maybe the line icon can also be considered a plain icon? 
If so, you can change the filename to `hadoop-plain.svg` and add this:
```suggestion
  	"aliases": [
            {
                "base": "plain",
                "alias": "line"
            }
        ] 
```